src/ui/tray-icon.cpp

```diff
@@ -352,13 +346,11 @@ QIcon SeafileTrayIcon::stateToIcon(TrayState state)
 #endif
 }
 
-void SeafileTrayIcon::toggleMainWindow()
+void SeafileTrayIcon::showMainWindow()
 {
     MainWindow *main_win = seafApplet->mainWindow();
     if (!main_win->isVisible()) {
```

Comment:
This if condition should be removed, otherwise seafile applet won't be brought to front when the focus is on anthor application. E.g. start seafile applet -> click somewhere else -> click "show main window" => nothing happens